pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,24 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting node NotReady
+	// condition. The amount of time when NodeController start evicting pods is controlled
+	// via flag 'pod_eviction_timeout'. Note: be cautious when changing nodeMonitorGracePeriod,
```

Comment:
I assume pod_eviction_timeout starts counting from when the node becomes NotReady? (For example, if pod_eviction_timeout is 2 * time.Second, then a pod will be evicted 10 seconds after the node becomes NotReady?). If so, then I would change this sentence to "Once node becomes NotReady, NodeController will start evicting pods after the additional amount of time specified by the flag 'pod_eviction_timeout'."